packages/manager/apps/web/client/app/hosting/hosting.constant.js

```diff
@@ -21,4 +21,17 @@ angular
     DELETING: 'deleting',
     REGENERATING: 'regenerating',
     UPDATING: 'updating',
-  });
+  })
+  .constant('HOSTING_UPGRADES', [
```

Comment:
can't you fetch this enum from the API ?

Reply:
Good point !